crates/rpc/rpc-types/src/eth/trace/common.rs

```diff
@@ -8,7 +9,7 @@ use serde::{Deserialize, Serialize};
 #[allow(missing_docs)]
 pub enum TraceResult<Ok, Err> {
     /// Untagged success variant
-    Success { result: Ok },
+    Success { result: Ok, tx_hash: TxHash },
     /// Untagged error variant
     Error { error: Err },
```

Comment:
If we look at the Go variant of that:

https://github.com/ethereum/go-ethereum/pull/27183/files#diff-bd288cd51c1874c7e65ce0d35047c0a98366479c8924606c9efc577f6a98f75c

it seems that the transaction hash information should also be added to the `Error` variant.

Reply:
correct, missed this comment—we need to add it to the error variant too